frontend/src/pages/GraphPF/components/node.tsx

```diff
@@ -38,15 +38,16 @@ import {
 } from '@patternfly/react-topology/dist/esm/components/nodes/NodeShadows';
 import { PFColors } from 'components/Pf/PfColors';
 import { kialiStyle } from 'styles/StyleUtils';
+import { keyframes } from 'typestyle';
 
 // This is a copy of PFT DefaultNode (v4.68.3), then modified.  I don't see a better way to really
 // do this because DefaultNode doesn't really seem itself extensible and to add certain behavior you have
 // to reimplement the rendered element.  This supports the following customizations:
-//   [Node] element.data.isFind?: boolean                // adds graph-find overlay
-//   [Node] element.data.isHighlighted?: boolean         // adds highlight effects based on hover
-//   [Node] element.data.isUnhighlighted?: boolean       // adds unhighlight effects based on hover
-//   [Node] element.data.hasSpans?: Span[]               // adds trace overlay
-//   [NodeLabel] isHover                                 // adds "raise" logic to bring label to the top
+//   [Node] isFind?: boolean                // adds graph-find overlay
+//   [Node] isUnhighlighted?: boolean       // adds unhighlight effects based on hover
+//   [Node] hasSpans?: Span[]               // adds trace overlay
+//   [Node] isFocused?: boolean             // adds focus overlay
```

Comment:
```suggestion
//   [Node] isFind?: boolean                // adds graph-find overlay
//   [Node] isFocused?: boolean             // adds focus overlay
//   [Node] isUnhighlighted?: boolean       // adds unhighlight effects based on hover
//   [Node] hasSpans?: Span[]               // adds trace overlay
```

Sorry to be a pain in the butt, but when a list is already alphabetized let's keep it that way.